python/pyspark/sql/session.py

```diff
@@ -230,7 +230,10 @@ def __init__(self, sparkContext, jsparkSession=None):
             SparkSession._instantiatedSession = self
             SparkSession._activeSession = self
             self._jvm.SparkSession.setDefaultSession(self._jsparkSession)
-            self._jvm.SparkSession.setActiveSession(self._jsparkSession)
+            self._jvm.java.lang.Class.forName("org.apache.spark.sql.SparkSession$")\
```

Comment:
`Class.forName` should better not directly used. This is banned by Scala style:

https://github.com/apache/spark/blob/e93b8f02cd706bedc47c9b55a73f632fe9e61ec3/scalastyle-config.xml#L197-L206